client/app/bundles/HelloWorld/actions/helloWorldActionCreators.jsx

```diff
@@ -0,0 +1,8 @@
+import actionTypes from '../constants/helloWorldConstants';
+
+const updateName = (text) => ({
+  type: actionTypes.HELLO_WORLD_NAME_UPDATE,
+  text,
+});
+
+export default updateName;
```

Comment:
Here if I wrote the following code, the linter would warn that I should use `export default` instead
```JavaScript
export const updateName = (text) => ({
  type: actionTypes.HELLO_WORLD_NAME_UPDATE,
  text,
});
```